README.md

```diff
@@ -1 +1,110 @@
-# BC Vaccine Card-Android
\ No newline at end of file
+# BCVAX-Android
```

Comment:
Change to 
BC Vaccine Card Verifier-Android

Reply:
changed